### PR TITLE
[T3A3][F10-A1]Joel Foo

### PR DESCRIPTION
**Pull request by jaeoheeail**
Ready for review.

### CODE REVIEW

File: test/expected.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@
 || Enter command: || [Command entered:  sfdfd]
 || add: Adds a person to the address book. Contact details can be marked private by prepending 'p' to the prefix.
 || Parameters: NAME [p]p/PHONE [p]e/EMAIL [p]a/ADDRESS  [t/TAG]...
-|| Example: add John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney
+|| Example: add John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25, 231534 t/friends t/owesMoney
```
Comment (Candiie) [inline test/expected.txt]: Well done on updating Test! Well done for separating the commit !

File: src/seedu/addressbook/data/person/Address.java [hunk 2]
```diff
@@ -67,7 +69,7 @@ public static boolean isValidAddress(String test) {
      * Returns true if a given string is a valid person address.
      */
     private boolean isValidAddressFields(String[] addressFields) {
-    	if (addressFields.length == 4) {
+    	if (addressFields.length == NUM_ADDRESS_FIELDS) {
```
Comment (Candiie) [inline src/seedu/addressbook/data/person/Address.java]: Good Catch!

File: src/seedu/addressbook/data/person/Address.java [hunk 3]
```diff
@@ -21,6 +21,8 @@
     private static final int ADDRESS_UNIT_INDEX = 2;
     private static final int ADDRESS_POSTAL_CODE_INDEX = 3;
     
+    private static final int NUM_ADDRESS_FIELDS = 4;
```
Comment (Candiie) [inline src/seedu/addressbook/data/person/Address.java]: Nice Job Joel, for extracting the magic numbers!

**Comment by Candiie** [comment]
Some comments added. Please close the PR after reading comments :+1

:+1: Hi Joel ! Well done, simple and clean execution, however for this round of execution since this task is simple it is ok to do the changes and push all in 1 commit (for the classes implementation) however, in future commits do take note to commit after each significant changes instead of all in one :) (e.g. splitting classes implementations into different commits.

**Comment by jaeoheeail** [comment]
Will take note for future commits! Will try to see if I can rectify my commits for T4 (which I have yet to perform a PR).

**Pull request closed without merging**